tests/core/RequestHandler.spec.ts

```diff
@@ -136,6 +138,13 @@ describe('RequestHandler', () => {
     await batchScheduler.writeOperationBatch();
   });
 
+  it('should resolve long form did from test vectors correctly', async () => {
```

Comment:
Sorry I wasn't being specific, I was more mainly looking for (still do) you to add a test vector for the new did suffix compute algorithm:

search for `it('should pass test vector.', async (done) => {`